### ISSUE TITLE
https://knot-design.jp/MODALit/ is gone

### ISSUE THREAD

**Issue by yangar** [knot-design/MODALite#7]
Hi,

Is there a way to have access to the documentations on https://knot-design.jp/MODALit/ ?
The website seem to be gone.

Thank you!

**Comment by cinzianicoletti** [comment]
Yes I confirm, the site has not been reached for some time. Is there a way to view the documentations somewhere?
Can you put the latest version of the site in the folder demo?

Please :(

**Comment by yangar** [comment]
The documentations can be reached here:
https://web.archive.org/web/20180506180943/knot-design.jp/MODALit/
